controllers/storagecluster/cephcluster.go

```diff
@@ -1101,6 +1101,7 @@ func generateMgrSpec(sc *ocsv1.StorageCluster) rookCephv1.MgrSpec {
 		Modules: []rookCephv1.Module{
 			{Name: "pg_autoscaler", Enabled: true},
 			{Name: "balancer", Enabled: true},
+			{Name: "rook", Enabled: false},
```

Comment:
Are we planning to keep it disabled permanently? I doubt 🤔

Reply:
Technically it is supposed to be disabled when it is installed. But due to some issues as observed in this bz: https://bugzilla.redhat.com/show_bug.cgi?id=2300312#add_comment the rook mgr module is enabled initially. This just makes sure that on installation it remains disabled. Can be enabled through the rook toolbox post installation.

> Are we planning to keep it disabled permanently? I doubt 🤔

Reply:
Understood, but what if in future we would want to enable the module for some case? By disabling it this way, there would be clash between a user manually enabling it through toolbox and the operator reconciling it to disable.
@travisn is it guaranteed that in future as well we don't see any use case for rook-mgr module in ODF d/s?

Reply:
The `rook` mgr module is really only used by the upstream dashboard, which we don't use downstream. There is not currently a scenario to enable this module in the downstream. When it is enabled (though we are not sure the cause), it causes errors as we have seen in a couple customer clusters.